src/main/java/org/opentripplanner/routing/graph/index/StreetIndex.java

```diff
@@ -138,7 +139,8 @@ public VertexLinker getVertexLinker() {
     return vertexLinker;
   }
 
-  public TransitStopVertex findTransitStopVertices(FeedScopedId stopId) {
+  @Nullable
+  public TransitStopVertex getTransitStopVertex(FeedScopedId stopId) {
```

Comment:
FYI: According to the naming conventions this does not follow either [`get` or `find`](https://github.com/opentripplanner/OpenTripPlanner/blob/dev-2.x/CODE_CONVENTIONS.md#methods). Changing this to throw an exception is the "right" thing to do, but I am not sure if we can do it - we might have stops outside the area covered by OSM data - and throwing exception should only be done in cases where we expect it to not happen. I would not have changed the name here.

Reply:
Reverted in d91d7c17f31f571433156d5a12a9a7ce2a498c92 but the name `findTransitStopVertices` is obviously wrong.